tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "target": "ES2020",
+    "module": "ES2020",
```

Comment:
Not sure about how this changes compilation, but it's required for async imports

Reply:
This will compile the project into ESM
We had talked about it preivously but decided not to migrate to ESM for some reasons. Im not sure if if it's still up to date now

However if we have to stay in CommonJS we can use `require` instead of `await import`.

Reply:
Yes, @Julien-R44 is still right here, we need to support CJS and ESM
That's why there are two separate js files
- src/index.js
- src/index.mjs